### PR TITLE
Update/v0.4.1

### PR DESCRIPTION
**Pull request by kn0wmad**

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -55,8 +44,8 @@ config:
     type: script
   set:
     type: script
-properties: 
-  type: script
+# properties:
```
Comment (islandbitcoin) [inline manifest.yaml:47]: why did you remove this?

File: scripts/embassy.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
 export { setConfig } from "./procedures/setConfig.ts";
 export { getConfig } from "./procedures/getConfig.ts";
-export { properties } from "./procedures/properties.ts";
+// export { properties } from "./procedures/properties.ts";
```
Comment (islandbitcoin) [inline scripts/embassy.ts:3]: same question as above

Comment (kn0wmad) [reply scripts/embassy.ts:3]: Properties were trying to load - but there aren't any

**Review by islandbitcoin — APPROVED** [review]
LGTM
